test/assignment5/sockettest.sh

```diff
@@ -71,6 +71,40 @@ function test_send_socket_string
 	fi
 }
 
+# Test that sends multiple packets at once
+# @param1 : The string to send
+# @param2 : The previous compare file
+# Returns if the test passes, exits with error if the test fails.
+function test_send_multiple_packets
```

Comment:
@DeekshithPatil I'd like to avoid duplicating logic here.  Can we modify the send string function to support sending more than one string if it doesn't support this already?  I think you might be able to just specify newlines in the argument.  Did you try this?